quay.io/pawsey/openfoam-org/aliases/10.yaml

```diff
@@ -0,0 +1,186 @@
+aliases:
+  PDRFoam: PDRFoam
```

Comment:
Aliases should always be full paths

Reply:
Thanks a lot @vsoch.
The *.yaml files for the aliases now refer to their full paths. Not only for `10.yaml` but for all the versions.